.github/workflows/default_on_push.yml

```
@@ -63,7 +63,7 @@ jobs:
           submodules: recursive
       - name: Prepare tags
         id: prep
-        uses: crazy-max/ghaction-docker-meta@v2.5.0
+        uses: crazy-max/ghaction-docker-meta@v3
```

```suggestion
        uses: docker/metadata-action@v3
```

The repo moved to the docker org so not only the version but the whole repo needs to be changed